docs/integrations/microsoft-azure/azure-key-vault.md

```diff
@@ -33,24 +33,97 @@ When you configure the event hubs source or HTTP source, plan your source catego
 
 ### Configure metrics collection
 
-In this section, you will configure a pipeline for shipping metrics from Azure Monitor to an Event Hub, on to an Azure Function, and finally to an HTTP Source on a hosted collector in Sumo Logic.
+:::note
+Sumo Logic Metrics source is currently in Beta, to participate, contact your Sumo Logic account executive.
+:::
+
+In the Sumo Logic Azure Metrics source configuration,
+
+- Configure namespaces as `Microsoft.KeyVault/vaults` and `Microsoft.KeyVault/managedhsms`. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/azure-key-vaults-namespaces.png')} alt="Azure Key Vaults Namespaces" style={{border: '1px solid gray'}} width="500" />
 
-1. [Configure an HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
-2. [Configure and deploy the ARM Template](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-2-configure-azure-resources-using-arm-template).
-3. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Key Vault that you want to monitor.
```

Comment:
1. To set up the Azure Metrics source in Sumo Logic, refer to the shared beta documentation.
1.  In the Sumo Logic Azure Metrics source configuration, configure namespaces as `Microsoft.KeyVault/vaults` and `Microsoft.KeyVault/managedhsms`. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/azure-key-vaults-namespaces.png')} alt="Azure Key Vaults Namespaces" style={{border: '1px solid gray'}} width="500" />